CHANGELOG-0.2.md

```diff
@@ -1,5 +1,20 @@
 # Changelog 0.2
 
+## [0.2.2] 2019-03-27
+
+### Added
+
+- Documentation for manual upgrade Kubernetes from 1.13.0 to 1.13.1 [link](./docs/home/HOWTO.md#how-to-upgrade-kubernetes-cluster-from-1.13.0-to-1.13.1)
+
+### Changed
+- Kubernetes version 1.13.1 #178
+- Docker version for RedHat to 18.06
+
+### Fixed
+
+- Fix issue with kubernetes-cni in 0.2.2 #177
```

Comment:
This is changelog for v0.2.2, so I think it should rather contain info what has been changed/fixed compared to the previous versions, so maybe this line is not informative.